lib/rspec/core/configuration.rb

```diff
@@ -471,6 +473,9 @@ def initialize
       # Used to set higher priority option values from the command line.
       def force(hash)
         ordering_manager.force(hash)
+        [:deprecation_stream, :output_stream].each do |stream|
+          hash[stream] = prepare_stream(hash[stream]) if hash.key?(stream)
```

Comment:
Yep if the intent is to pass a modified hash down to ordering_manager, use merge on the hash as it's passed in e.g.

```
ordering_manager.force(hash.merge(
...
))
```